### ISSUE TITLE
All functions (buttons) in right click context menu

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#602]
**Reported by rebirther on 23 Mar 38312230 21:46 UTC**
The BM is much too wide, to improve user activity (abort, suspend, resume etc.) it is better to handle with right mouse click, you remembering on 4.19?

Migrated-From: http://boinc.berkeley.edu/trac/ticket/624

**Comment by Ageless93** [comment]
Is this still something that people yearn for? 
Is it easier to use command buttons, or to use right-click context menus?

**Comment by lfield** [comment]
I don't think this is an issue.

**Comment by AenBleidd** [comment]
It is not an issue but nice feature request.
I reopen it to implement it in future

**Comment by CharlieFenton** [comment]
> nice feature request.

I strongly disagree. 

* Implementing functionality with right clicks instead of buttons makes it harder to find or even be aware of those features. I oppose all such "hidden functionality."

* This is a very Windows-centric request. Although Macs can simulate a right-click by holding the _control_ key while clicking the mouse, many Mac mice don't have a second button. And the control-click on Apple software  is generally reserved for advanced functions. Yes, I know that Linux tends to use right clicks also.

**Comment by CharlieFenton** [comment]
> The BM is much too wide

That was written Feb 3, 2015. Wider computer monitors are significantly more affordable anymore common now.

**Comment by Ageless93** [comment]
I must say that I have a lot of extra unused space now with my 3840x2160p in BOINC Manager, even when I don't use it full screen. I'm not sure how we can fix that, other than perhaps setting both the Manager and the Event Log to a fixed width.

**Comment by AenBleidd** [comment]
> I strongly disagree.

Ok, this can be discussed but from my POV right click context menu is a common approach and it is easier to use it rather than moving mouse pointer on big screen

**Comment by RichardHaselgrove** [comment]
> setting both the Manager and the Event Log to a fixed width.

No, please don't do that. We all have our own differing preferences for how our screens should be arranged, and the mechanism for remembering where and what size each component was last placed seems to work well (until the screen geometry changes). On machines where I'm not doing other work, I like to set up BOINC to use the whole screen, with the Event Log at minimum width on the right, and the main Manager display to the left. Not everyone will want that, but the option should be there.

**Comment by truboxl** [comment]
I think I will left my opinion here. Right click contexts menu doesn't make sense on touchscreen, eg: you don't do that on Android except there you hold it down. The current UI works perfectly as it has single touch buttons.

**Comment by truboxl** [comment]
Windows 10 comes with Tablet mode so maybe can use that to emulate it for testing the PR if it really works.

**Comment by AenBleidd** [comment]
@truboxl, this ticket is about Desktop mode only
